package.json

```diff
@@ -430,6 +430,16 @@
             "type": "boolean",
             "default": false,
             "markdownDescription": "Whether to use full path to integrated runme executable when running CLI commands. This is mostly useful for development purposes."
+          },
+          "runme.app.apiUrl": {
+            "type": "string",
+            "default": "https://app.runme.dev/graphql",
```

Comment:
@degrammer let's make the default no value please until this is closer to final.

Reply:
Fixed!